fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -532,16 +537,52 @@ public List<Expression> visitNamedExpressionSeq(NamedExpressionSeqContext namedC
      * @return List of OrderKey
      */
     @Override
-    public List<OrderKey> visitQueryOrganization(QueryOrganizationContext ctx) {
```

Comment:
nit: we could just add sort node and limit node to plan in withQueryOrganization directly and remove this function and related data structure like LimitAndOffset..

Reply:
Consider this function: `public LimitAndOffset visitLimitClause(LimitClauseContext ctx)`
If we remove `LimitAndOffset`, we still need a container to encapsulates `limit` and `offset` as a return result. 
Maybe we could use `Pair` instead of `LimitAndOffset`